iwyu.cc

```diff
@@ -898,21 +898,13 @@ class BaseAstVisitor : public RecursiveASTVisitor<Derived> {
         const_cast<CXXDestructorDecl*>(dtor), parent_type, expr);
   }
 
-  // This is to catch assigning template functions to function pointers.
+  // This is to catch binding of function pointers to templates.
```

Comment:
I find this sentence confusing. After some thought I understood what you mean but it took time. One of the biggest problems is how "bind" is used. It is used both "bind a variable to a value" and "bind data to parameter", i.e. direction of binding is ambiguous. And do we care if actual binding takes place? For example, in `&TplFn<MyClass*>;`? I am thinking about a comment like "This is to handle function pointers to template functions."